use_case_examples/llm/utility_functions.py

```diff
@@ -32,7 +32,7 @@ def max_fhe_relu(q_x, axis=-1, keepdims=True):
         if keepdims:
             shape = list(result.shape)
             shape.insert(axis, 1)
-            result = result.reshape(shape)
+            result = result.reshape(tuple(shape))
```

Comment:
is this change expected ? 🤔

Reply:
it's just an additional fix that could have used a separate PR but got stuck in this one

Reply:
got it thanks